tools/gitlint-extra-rules.py

```diff
@@ -40,7 +40,8 @@ def validate(self, commit: Any) -> Optional[List[RuleViolation]]:
         error = ("Areas at start of title should be lower case{}, "
                  "followed by ': '".format(exclusions_text))
         for area in title_components[:-1]:
-            if not (area.islower() or area in exclusions) or ' ' in area:
+            if (any(word not in exclusions for word in area.split('/')
+                    if word.isupper()) or ' ' in area):
```

Comment:
This logic isn't quite right, as it lets through eg. "Foo/keys/Readme: Something."

Reply:
@neiljp Updated